src/patternfly/components/Backdrop/styles.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../patternfly-utilities";
+
+:root {
+  --pf-c-backdrop--ZIndex: var(--pf-global--ZIndex--xl);
+  --pf-c-backdrop--Color: rgba(0, 0, 0, .5);
+  --pf-c-backdrop--BackdropFilter: blur(10px);
+}
+
+.pf-c-backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: var(--pf-c-backdrop--ZIndex);
+  width: 100%;
+  height: 100%;
+  background-color: var(--pf-c-backdrop--Color);
+  // Vendor prefix needed for Safari, but it is not getting added in the autoprefixer right now
+  //-webkit-backdrop-filter: var(--pf-c-backdrop--BackdropFilter);
```

Comment:
If autorpefixer is not prefixing it, I doesn't hurt to uncomment line 18

Reply:
It then fails to build because of the linter.